virttest/qemu_vm.py

```diff
@@ -2302,11 +2293,17 @@ def sort_key(dev):
             # disable-legacy and disable-modern options explicitly.
             set_disable_legacy = params.get("set_disable_legacy", "no")
             set_disable_modern = params.get("set_disable_modern", "no")
+            set_iommu_platform = params.get("set_iommu_platform", "no")
```

Comment:
I'd suggest using only one param to represent these features.

Given option `foo` and its value `bar`:
a. set `foo=on` if the value is `on`
b. set `foo=off` if the value is `off`
c. do not set the option if the value is undefined (`None`)

Reply:
@luckyh  thanks, updated, could you help review it again ?
And the reference will be updated in another patch later.